### PR TITLE
Add advice about not worrying about timelines

### PR DESCRIPTION
**Pull request by I3uckwheat**
I've added some advice about timelines to the getting started stuff that keeps getting iterated over and over and over in the Discord. Maybe this will help answer some of the questions before getting to the chat room.

### CODE REVIEW

File: web_development_101/the_basics/gearing_up.md [hunk 1]
```diff
@@ -74,6 +74,14 @@ You will inevitably get stuck at some point in the curriculum, perhaps due to a
...
+
+Deadlines cause un-needed stress. Since TOP is a free platform, that does not adhere you to a deadline, creating your own deadlines is a good way to rush concepts that should not be rushed. TOP is very research based, meaning you will have to do research to complete tasks and projects. There's no telling if you can find the article or post that helps you in the right way, quickly to meet your deadlines, but I bet you learned a TON along the way that you can use in the future. People that do this kind of research and strive to write better solutions tend to become better developers in the future. There's no knowing how long it could take you to learn how to query stuff to find your answers. There's no solid guidelines on that. If you're doing TOP because you need a high paying job right-now, you're likely not going to become a solid developer within the timeframe you have set. Stress and anxiety absolutely do not help you learn either. Relax and just enjoy the ride.
```
Comment (xandora) [inline web_development_101/the_basics/gearing_up.md:81]: "There's no solid guidelines on that." -> "There **are** no solid guidelines on that."

Comment (I3uckwheat) [reply web_development_101/the_basics/gearing_up.md:81]: @xandora Thank you :+1:

File: web_development_101/the_basics/gearing_up.md [hunk 2]
```diff
@@ -74,6 +74,14 @@ You will inevitably get stuck at some point in the curriculum, perhaps due to a
...
 
+It may take you longer than others to grasp concepts, or it may take you less time. This doesn't mean you're smarter or dumber than others, it means you've had differing life experiences that may or may not have prepared you for learning this stuff. Someone who grew up around an engineer may have some advantages over someone who didn't, but it doesn't mean you can't learn those skills.
+
+The Odin Project isn't like college or university, TOP is self paced and allows you to get a really solid grasp of concepts before moving on. In school, you're forced to keep up or you will fail. The difference here is that coming into TOP, you're not expected to have much knowledge, there's no prerequisites. We've had people be successful coming through here that only knew how to check their email with a computer, we've also seen success from computer science degree holders. Treating TOP like a static timeline is ignorant simply because you don't know what you don't know yet, and that's OK! There's no due-dates on things in The Odin Project so you can spend the time to do it right and discuss the topics.
```
Comment (ZachBaird) [inline web_development_101/the_basics/gearing_up.md:79]: > TOP
I would reword anywhere you have the TOP acronym, just because I haven't seen it used in the curriculum anywhere else. If I'm mistaken, disregard.
> We've had people be successful coming through here that only knew how to check their email with a computer, we've also seen success from computer science degree holders
I would either make these separate sentences or add an "and" after the comma. If you don't like that flow, changing the comma to a semi-colon fixes the grammatical issue.
> is ignorant simply
I'm not sure what it would be just yet, but I think just using an alternate word with more positive connotations would be better here.

File: web_development_101/the_basics/gearing_up.md (hunk 1 above)
Comment (ZachBaird) [inline web_development_101/the_basics/gearing_up.md]: > that does not adhere you
I think you want to go with something like "you are not beholden to a timeline"
> you're likely not going to become a solid developer
Just take out the likely; this needs to be a firm point

**Review by ZachBaird — COMMENTED** [review]
Left some thoughts I had reading through it

File: web_development_101/the_basics/gearing_up.md (hunk 2 above)
Comment (I3uckwheat) [reply web_development_101/the_basics/gearing_up.md:79]: I have fixed the first two points, as for the last one:

> Treating The Odin Project like a static timeline is understandable, but is a sign of misplaced expectations, simply because you don't know what you don't know yet, and that's OK!

Does this sound OK?

File: web_development_101/the_basics/gearing_up.md (hunk 1 above)
Comment (I3uckwheat) [reply web_development_101/the_basics/gearing_up.md]: Fixed! Let me know if there needs to be more changes

File: web_development_101/the_basics/gearing_up.md (hunk 2 above)
Comment (ZachBaird) [reply web_development_101/the_basics/gearing_up.md]: I like it, the sentence is just too long. I think breaking it up into different ones like

> Treating The Odin Project like a static timeline is understandable, but is a sign of misplaced expectations. You simply don't know what you don't know yet, and that's OK!

Reads a little better.

File: web_development_101/the_basics/gearing_up.md (hunk 1 above)
Comment (ZachBaird) [reply web_development_101/the_basics/gearing_up.md]: > , creating your own deadlines is a good way to rush concepts that should not be rushed.

I'd make this into its own sentence.

File: web_development_101/the_basics/gearing_up.md (hunk 2 above)
Comment (I3uckwheat) [reply web_development_101/the_basics/gearing_up.md]: Done.

File: web_development_101/the_basics/gearing_up.md (hunk 1 above)
Comment (I3uckwheat) [reply web_development_101/the_basics/gearing_up.md]: Done!
